src/seql/helpers.clj

```diff
@@ -134,32 +135,87 @@
    :spec    spec
    :handler `(fn ~bindv ~@body)})
 
+(defn add-create-mutation
+  "Add a mutation to insert a new record in the database.
+   By default the mutation will be named `create` in the
+   corresponding's entity namespace.
+
+   The mutation needs a spec of its input."
+  ([]
+   (add-create-mutation :create))
+  ([create-kw]
+   {:type    :create-mutation
+    :context ::entity
+    :spec    create-kw}))
+
+(defn add-update-by-id-mutation
+  "Add an update mutation to modify a record in the database.
+   Needs an ident, which will be used to specify the target.
```

Comment:
The concept of `ident` (from a user perspective at least, because used extensively in the source code) is really fuzzy now we deprecated the usage of `ident`.
I would use the term `identity` which is more common.